stdlib/logging/config.pyi

```diff
@@ -50,17 +50,15 @@ _FilterConfiguration: TypeAlias = _FilterConfigurationTypedDict | dict[str, Any]
 # Handler config can have additional keys even when not providing a custom factory so we just use `dict`.
 _HandlerConfiguration: TypeAlias = dict[str, Any]
 
-class _OptionalDictConfigArgs(TypedDict, total=False):
-    formatters: dict[str, _FormatterConfiguration]
-    filters: dict[str, _FilterConfiguration]
-    handlers: dict[str, _HandlerConfiguration]
-    loggers: dict[str, _LoggerConfiguration]
-    root: _RootLoggerConfiguration
-    incremental: bool
-    disable_existing_loggers: bool
-
-class _DictConfigArgs(_OptionalDictConfigArgs, TypedDict):
+class _DictConfigArgs(TypedDict):
     version: Literal[1]
+    formatters: NotRequired[dict[str, _FormatterConfiguration]]
+    filters: NotRequired[dict[str, _FilterConfiguration]]
+    handlers: NotRequired[dict[str, _HandlerConfiguration]]
+    loggers: NotRequired[dict[str, _LoggerConfiguration]]
+    root: NotRequired[_RootLoggerConfiguration]
+    incremental: NotRequired[bool]
+    disable_existing_loggers: NotRequired[bool]
```

Comment:
Ugh... maybe we should just wait for pytype to fix the `Required` bug? I agree the existing hack is ugly, but this seems pretty ugly too :(

Reply:
While this is also ugly, I still prefer it to the old hack, which is a bit hard to understand. That said, maybe the pytype problem gets fixed quickly.

Reply:
Yeah, this should be a quick fix in pytype =) I'll try to get a release out in the next day or two.

Reply:
Pytype release 2023.11.29 contains the fix.